### ISSUE TITLE
Station Wheeler (2R018) to SB I-5

### ISSUE THREAD

**Issue by chimai2027** [adus/portal-documentation#8]
We can't find ramp volume for 4/23/2019.  Can you please check for availability and pull for us?  Thanks!
Also, I pulled data for a couple dates in May and the ramp volumes appear too high.  Please see attached file.
[2R018_RampData.xlsx](https://github.com/adus/portal-documentation/files/3308424/2R018_RampData.xlsx)

**Comment by kristinann** [comment]
@basemelazzabi  Can you check on this? Thank you.

**Comment by basemelazzabi** [comment]
@chimai2027, For 4/23/2019, I checked the raw data that we have and it seems that we received data but it shows the volume to be zero for the entire day. I can't tell from the data if this is the correct observation for that day or if the detector sent wrong data. @chimai2027, If you want, I can send the full raw data that we have for that day. There are other fields in the raw data whose values I don't know what they mean.

As for the high volume issue, I also checked the raw data, and it seems that the one hour aggregations are correct. Which means there is a problem with the raw data itself. I attached the file that contains the raw data. I only included the volume and timestamp fields. If you want to see the other fields, let me know.

[ramp_data.zip](https://github.com/adus/portal-documentation/files/3315673/ramp_data.zip)
